### PR TITLE
Add get speakers

### PR DESCRIPTION
**Pull request by jebbinSCL**
Renaming Speaker and Admin Proxy to * Facade because they only delegate calls to multiple proxies. 
Add /Speaker GET

### CODE REVIEW

File: Gateway/SpeakerFacade.fs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+let getSpeaker speakerId = 
+    let profile = getProfile speakerId 
+    let handlesDtos = getHandlesByProfileId speakerId 
+    Profile.toSpeaker handlesDtos profile
+
+let getSpeakers () = 
+    let profiles = getProfiles() 
+    profiles |> Seq.map (fun profile -> 
+        let handlesDtos = getHandlesByProfileId profile.Id 
+        Profile.toSpeaker handlesDtos profile)
```
Comment (pallenSL) [inline Gateway/SpeakerFacade.fs]: Repetition.  I thought F# made that a thing of the past?  :D

Comment (jebbinSCL) [inline Gateway/SpeakerFacade.fs]: The repetition is the 2 lines which get the handles for the profile and converting to a speaker. I prefer the repetition to having the following function which would do the last 2 / 3 steps for getting a profile: 
``` 
let getProfileHandlesAndConvertToSpeaker (profile : Dtos.Profile) = 
    let handlesDtos = getHandlesByProfileId profile.Id 
    Profile.toSpeaker handlesDtos profile
```

**Comment by pallenSL** [comment]
Well, throw it overboard then!
:shipit:

**Comment by dmorrisSL** [comment]
:ship:

**Pull request merged**